src/custom/components/Version/index.tsx

```diff
@@ -5,11 +5,12 @@ import { ExternalLink, TYPE } from 'theme'
 import { GP_SETTLEMENT_CONTRACT_ADDRESS } from 'custom/constants'
 import { getEtherscanLink } from 'utils'
 
-import { version as WEB, dependencies } from 'src/../../package.json'
+import { version as WEB } from 'src/../../package.json'
```

Comment:
Absolute paths + backtracking :clap:

Reply:
Can do this, btw, `@src/../package.json'`

Reply:
lol